BraveWallet/Crypto/Accounts/AccountListView.swift

```diff
@@ -45,20 +48,28 @@ struct AccountListView: View {
           }
         }
         ToolbarItemGroup(placement: .primaryAction) {
-          Button(action: { isPresentingAddAccount = true }) {
+          Button(action: {
+            isPresentingCoinTypes = true
+          }) {
             Label(Strings.Wallet.addAccountTitle, systemImage: "plus")
               .foregroundColor(Color(.braveOrange))
           }
         }
       }
-      .sheet(isPresented: $isPresentingAddAccount) {
-        NavigationView {
-          AddAccountView(keyringStore: keyringStore)
-        }
-        .navigationViewStyle(StackNavigationViewStyle())
-      }
     }
     .navigationViewStyle(StackNavigationViewStyle())
+    .panModal(isPresented: $isPresentingCoinTypes) {
```

Comment:
I think you want this modifier on the Label in the button for add account:
<img width="783" alt="Screen Shot 2022-07-07 at 11 18 43 AM" src="https://user-images.githubusercontent.com/5314553/177810452-d1bcf3d2-cd76-45f3-bff0-8cdefae9c318.png">

Same for other usages 🙂.